pallets/api/src/fungibles/tests.rs

```diff
@@ -416,10 +417,17 @@ fn burn_works() {
 		let from = BOB;
 		let total_supply = value * 2;
 
-		// Check error works for `Assets::burn()`.
-		assert_noop!(Fungibles::burn(signed(owner), token, from, value), AssetsError::Unknown);
```

Comment:
Whilst the assets error assertions were to ensure that errors are properly handled, which is the case on line 424, this line could have been left in.